test/user.js

```diff
@@ -4,7 +4,30 @@ const chai = require('chai'),
   should = chai.should(),
   expect = chai.expect,
   User = require('./../app/models').User,
-  user = { name: 'name', lastname: 'lastname', email: 'test@wolox.co', password: '12345678' };
+  user = { name: 'name', lastname: 'lastname', email: 'test@wolox.co', password: '12345678' },
+  cookie = require('cookie'),
+  createUser = () =>
+    chai
+      .request(app)
+      .post('/users/')
+      .send(user);
+
+const loginUser = () =>
+  chai
+    .request(app)
+    .post('/users/sessions')
+    .send({ email: user.email, password: user.password });
+
+const getToken = res => {
+  const cookies = res.headers['set-cookie'];
+  let token;
+  if (cookies) {
+    cookies.forEach(function(icookie) {
```

Comment:
why a forEach?

Reply:
because maybe the header can have more than one cookie, so I iterate over the cookies and get the x-access-token cookie. And the library 'cookie' helps me to parse the cookie and get just the value that I want